src/solution.c

```diff
@@ -78,6 +80,9 @@ static u16 lock_counters[PLATFORM_SIGNAL_COUNT];
 bool disable_raim = false;
 bool send_heading = false;
 
+/* The flag is true when we have a fix */
+static bool soln_flag = false;
```

Comment:
This variable seems to be used only inside solution_thread loop. Wouldn't it be better to declare this variable inside the thread function instead of file scope global?